src/librustc_back/svh.rs

```diff
@@ -355,7 +345,7 @@ mod svh_visitor {
                     &MacInvocTT(ref path, ref _tts, ref _stx_ctxt) => {
                         let s = &path.segments;
                         assert_eq!(s.len(), 1);
-                        content(s[0].identifier)
+                        s[0].identifier.name.into()
```

Comment:
uhm yea... that was me trying to be "idiomatic"... then I noticed that it won't work when I need a `&str`... cleaning it up :)